ui/window.go

```diff
@@ -96,6 +98,7 @@ func CreateWindow(wal *wallet.Wallet) (*Window, *app.Window, error) {
 	win.invalidate = make(chan struct{}, 2)
 
 	win.wallet = wal
+
```

Comment:
Can also undo this change so this PR doesn't register any change to this file as it is not required.